notebooks/10_inference_in_bayesian_networks_1/index.md

```diff
@@ -0,0 +1,201 @@
+# Inference in Bayes Nets 1
+
```

Comment:
Well done with your Lecture Note! You've done a great job!

These few comments will help your LN get the best possible score:
- Consider resizing your images; Some of them are excessively large
- Your formulas are not correctly processed and represented by Github. For example, you've put $x_i$ and Github has put this exact thing on the final page. Lots of students have had this problem, reach out to them and fix it. Btw, I am visiting your LN via this link: https://github.com/Parsa2820/notes/blob/lecturenote/10_bayesNetInference/notebooks/10_inference_in_bayesian_networks_1/index.md
- If the texts get "Justified", then the article will look much better.

Reply:
Thanks for your review. We appreciate your suggestions.

- Images sizes were specified, but Github ignored them. I replaced the `div` HTML tag with `p`, which is now correctly shown.  
- The whole document is now justified, and yes, it looks much better!

About the second suggestion, Github does not support math equations (there is an open [issue](https://github.com/github/markup/issues/1294) for that), but Mr. Zehtab (@vahidzee ) have mentioned in Telegram that equations will be shown up in the Webifier after deployment. We have already tested the equations in our editor (VSCode), and they seem sound. By the way, if there is still a problem with them, we will be happy to solve them.